Sources/Bugsnag/PayloadTransformer.swift

```diff
@@ -12,24 +12,29 @@ public protocol PayloadTransformerType {
         metadata: Node?,
         request: Request?,
         severity: Severity,
-        stackTraceSize: Int,
-        filters: [String]
+        stackTraceSize: Int?,
```

Comment:
Why did you choose to make these optionals? I understand that it simplifies the callsite of `payLoadFor(..` (which is internal) but I guess the downside is that we know how some knowledge on fallback/default values two places (although set from the outside). Thoughts?